.commitlintrc.ts

```diff
@@ -0,0 +1,52 @@
+import { UserConfig } from '@commitlint/types';
+import { LOCALE_IDS } from './lib/constants/locales';
+import { CALENDAR_IDS } from './lib/constants/calendars';
+
+const empty = [null];
+const calendarScopes = [...CALENDAR_IDS, ...LOCALE_IDS, 'l10n', 'calendar'];
+
+const typesEnumScoped = {
+  // Updates to the automation or release process
+  ci: empty,
+  // Changes to build scripts
+  build: empty,
+  // Documentation only changes
+  docs: empty,
+  // Features, net-new functionality
+  feat: calendarScopes,
+  // Enhancing existing functionality
+  enh: calendarScopes,
+  // Bug fixes
+  fix: calendarScopes,
+  // Refactoring, no behavior changes
+  refactor: calendarScopes,
+  // Performance improvements
+  perf: calendarScopes,
+  // Add or correct tests
+  test: calendarScopes,
+  // Changes that affect
+  style: calendarScopes,
+};
+
+export default {
+  extends: ['@commitlint/config-conventional'],
+  formatter: '@commitlint/format',
+  helpUrl: 'https://github.com/romcal/romcal/blob/dev/docs/contribute-to-romcal.md#committing-changes',
```

Comment:
I’d personally move the `Comitting changes` to `docs`, maybe to [`docs/contribute-to-romcal.md`](https://github.com/romcal/romcal/blob/dev/docs/contribute-to-romcal.md), however, that is just a preference.

Reply:
uhhh, committing changes is in docs... I'm not linking to the readme, i'm linking to contribute-to-romcal.md

Reply:
My bad then, sorry.

Reply:
you're ok, it's a lot of docs!

Reply:
Or maybe I review the PR too quickly (and I have done it a 6" screen of my phone). :man_shrugging: